packages/calypso-products/src/constants/features.ts

```diff
@@ -287,6 +287,7 @@ export const WPCOM_FEATURES_LIVE_SUPPORT = 'live_support';
 export const WPCOM_FEATURES_MANAGE_PLUGINS = 'manage-plugins';
 export const WPCOM_FEATURES_NO_ADVERTS = 'no-adverts/no-adverts.php';
 export const WPCOM_FEATURES_NO_WPCOM_BRANDING = 'no-wpcom-branding';
+export const WPCOM_FEATURES_SCHEDULED_UPDATES = 'scheduled-updates';
```

Comment:
Let's continue to keep this list alphabetized.

Reply:
Ah, I thought it was a different list after `WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED`, thanks for catching this!